scripts/performance/benchmark-mv

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+import argparse
+import subprocess
+import os
+
+from benchmark_utils import get_benchmark_script, get_summarize_script
+from benchmark_utils import human_readable_to_bytes, summarize
+from benchmark_utils import backup, copy, clean
+
+
+def benchmark_mv(args):
+    if not os.path.exists(args.result_dir):
+        os.makedirs(args.result_dir)
+    command = 'aws s3 mv %s %s' % (
+        args.source, args.destination)
+    
+    if args.recursive:
+        command += ' --recursive'
+
+    if args.quiet:
+        command += ' --quiet'
+    else:
+        print(command)
+
+    backup_path = backup(args.source, args.recursive)
+
+    for i in range(args.num_iterations):
+        out_file = 'performance%s.csv' % i
+        out_file = os.path.join(args.result_dir, out_file)
+        benchmark_args = [
+            args.benchmark_script, command, '--output-file', out_file
+        ]
+        if not args.dry_run:
+            subprocess.check_call(benchmark_args)
+            copy(backup_path, args.source, args.recursive)
+    
+    if not args.dry_run:
+        summarize(args.summarize_script, args.result_dir)
+
+        if not args.no_cleanup:
+            clean(args.destination, args.recursive)
+        clean(backup_path, args.recursive)
+    
+
+def main():
+    parser = argparse.ArgumentParser()
```

Comment:
This looks like an exact duplicate in the other `benchmark-*` scripts.  Seems like a candidate for moving into `benchmark_utils`.